### ISSUE TITLE
singa-gpu Anaconda package has no Module class

### ISSUE THREAD

**Issue by tinyAdapter** [apache/singa#791]
I'm trying to test some examples of SINGA. However, when I ran the examples, the `singa-gpu` package throwed an error that it could not find Module class.

I reproduce this error by the following commands:

```bash
(base) user@xgpe3:~$ conda activate sg37
(sg37) user@xgpe3:~$ conda install -c nusdbsystem -c conda-forge singa-gpu=3.0.0
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 4.8.3
  latest version: 4.8.4

Please update conda by running

    $ conda update -n base -c defaults conda



## Package Plan ##

  environment location: /path/to/miniconda3/envs/sg37

  added / updated specs:
    - singa-gpu=3.0.0


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    importlib_metadata-1.7.0   |                0           3 KB  conda-forge
    zipp-3.1.0                 |             py_0          10 KB  conda-forge
    ------------------------------------------------------------
                                           Total:          13 KB

The following NEW packages will be INSTALLED:

  _openmp_mutex      conda-forge/linux-64::_openmp_mutex-4.5-1_llvm
  attrs              conda-forge/noarch::attrs-20.2.0-pyh9f0ad1d_0
  cudatoolkit        pkgs/main/linux-64::cudatoolkit-10.0.130-0
  cudnn              pkgs/main/linux-64::cudnn-7.6.5-cuda10.0_0
  deprecated         conda-forge/noarch::deprecated-1.2.7-py_0
  dnnl               nusdbsystem/linux-64::dnnl-1.1-build
  freetype           conda-forge/linux-64::freetype-2.10.2-he06d7ca_0
  future             conda-forge/linux-64::future-0.18.2-py37hc8dfbb8_1
  glog               conda-forge/linux-64::glog-0.3.5-hf484d3e_1001
  importlib-metadata conda-forge/linux-64::importlib-metadata-1.7.0-py37hc8dfbb8_0
  importlib_metadata conda-forge/noarch::importlib_metadata-1.7.0-0
  iniconfig          conda-forge/noarch::iniconfig-1.0.1-pyh9f0ad1d_0
  jpeg               conda-forge/linux-64::jpeg-9d-h516909a_0
  lcms2              conda-forge/linux-64::lcms2-2.11-hbd6801e_0
  libblas            conda-forge/linux-64::libblas-3.8.0-16_openblas
  libcblas           conda-forge/linux-64::libcblas-3.8.0-16_openblas
  libgfortran-ng     conda-forge/linux-64::libgfortran-ng-7.5.0-hdf63c60_16
  liblapack          conda-forge/linux-64::liblapack-3.8.0-16_openblas
  libopenblas        conda-forge/linux-64::libopenblas-0.3.9-h5ec1e0e_0
  libpng             conda-forge/linux-64::libpng-1.6.37-hed695b0_2
  libprotobuf        conda-forge/linux-64::libprotobuf-3.9.2-h8b12597_0
  libtiff            conda-forge/linux-64::libtiff-4.1.0-hc7e4089_6
  libwebp-base       conda-forge/linux-64::libwebp-base-1.1.0-h516909a_3
  llvm-openmp        conda-forge/linux-64::llvm-openmp-10.0.1-hc9558a2_0
  lz4-c              conda-forge/linux-64::lz4-c-1.9.2-he1b5a44_3
  more-itertools     conda-forge/noarch::more-itertools-8.5.0-py_0
  numpy              conda-forge/linux-64::numpy-1.16.5-py37h95a1406_0
  olefile            conda-forge/noarch::olefile-0.46-py_0
  onnx               conda-forge/linux-64::onnx-1.6.0-py37he1b5a44_0
  packaging          conda-forge/noarch::packaging-20.4-pyh9f0ad1d_0
  pillow             conda-forge/linux-64::pillow-7.2.0-py37h718be6c_1
  pluggy             conda-forge/linux-64::pluggy-0.13.1-py37hc8dfbb8_2
  protobuf           conda-forge/linux-64::protobuf-3.9.2-py37he1b5a44_1
  py                 conda-forge/noarch::py-1.9.0-pyh9f0ad1d_0
  pyparsing          conda-forge/noarch::pyparsing-2.4.7-pyh9f0ad1d_0
  pytest             conda-forge/linux-64::pytest-6.0.1-py37hc8dfbb8_0
  python_abi         conda-forge/linux-64::python_abi-3.7-1_cp37m
  singa              nusdbsystem/linux-64::singa-3.0.0-cudnn7.6.5_cuda10.0_py37
  singa-gpu          nusdbsystem/linux-64::singa-gpu-3.0.0-py37
  six                conda-forge/noarch::six-1.15.0-pyh9f0ad1d_0
  toml               conda-forge/noarch::toml-0.10.1-pyh9f0ad1d_0
  tqdm               conda-forge/noarch::tqdm-4.48.2-pyh9f0ad1d_0
  wrapt              conda-forge/linux-64::wrapt-1.12.1-py37h8f50634_1
  zipp               conda-forge/noarch::zipp-3.1.0-py_0
  zstd               conda-forge/linux-64::zstd-1.4.5-h6597ccf_2

The following packages will be UPDATED:

  libgcc-ng           pkgs/main::libgcc-ng-9.1.0-hdf63c60_0 --> conda-forge::libgcc-ng-9.3.0-h24d8f2e_16
  openssl              pkgs/main::openssl-1.1.1g-h7b6447c_0 --> conda-forge::openssl-1.1.1g-h516909a_1

The following packages will be SUPERSEDED by a higher-priority channel:

  _libgcc_mutex           pkgs/main::_libgcc_mutex-0.1-main --> conda-forge::_libgcc_mutex-0.1-conda_forge
  ca-certificates    pkgs/main::ca-certificates-2020.7.22-0 --> conda-forge::ca-certificates-2020.6.20-hecda079_0
  certifi               pkgs/main::certifi-2020.6.20-py37_0 --> conda-forge::certifi-2020.6.20-py37hc8dfbb8_0


Proceed ([y]/n)? y


Downloading and Extracting Packages
importlib_metadata-1 | 3 KB      | #################################################################################### | 100%
zipp-3.1.0           | 10 KB     | #################################################################################### | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
(sg37) user@xgpe3:~$ python -c "from singa.module import Module"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'singa.module'
(sg37) user@xgpe3:~$
```

By contrast, with `singa-cpu` package, everything seems fine.

```bash
(base) user@xgpe3:~$ conda activate sc37
(sc37) user@xgpe3:~$ conda install -c nusdbsystem -c conda-forge singa-cpu=3.0.0
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 4.8.3
  latest version: 4.8.4

Please update conda by running

    $ conda update -n base -c defaults conda



## Package Plan ##

  environment location: /path/to/miniconda3/envs/sc37

  added / updated specs:
    - singa-cpu=3.0.0


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    singa-3.0.0                |         cpu_py37        22.2 MB  nusdbsystem
    singa-cpu-3.0.0            |             py37           4 KB  nusdbsystem
    ------------------------------------------------------------
                                           Total:        22.2 MB

The following NEW packages will be INSTALLED:

  _openmp_mutex      conda-forge/linux-64::_openmp_mutex-4.5-1_llvm
  deprecated         conda-forge/noarch::deprecated-1.2.7-py_0
  dnnl               nusdbsystem/linux-64::dnnl-1.1-build
  freetype           conda-forge/linux-64::freetype-2.10.2-he06d7ca_0
  future             conda-forge/linux-64::future-0.18.2-py37hc8dfbb8_1
  glog               conda-forge/linux-64::glog-0.3.5-hf484d3e_1001
  jpeg               conda-forge/linux-64::jpeg-9d-h516909a_0
  lcms2              conda-forge/linux-64::lcms2-2.11-hbd6801e_0
  libblas            conda-forge/linux-64::libblas-3.8.0-16_openblas
  libcblas           conda-forge/linux-64::libcblas-3.8.0-16_openblas
  libgfortran-ng     conda-forge/linux-64::libgfortran-ng-7.5.0-hdf63c60_16
  liblapack          conda-forge/linux-64::liblapack-3.8.0-16_openblas
  libopenblas        conda-forge/linux-64::libopenblas-0.3.9-h5ec1e0e_0
  libpng             conda-forge/linux-64::libpng-1.6.37-hed695b0_2
  libprotobuf        conda-forge/linux-64::libprotobuf-3.9.2-h8b12597_0
  libtiff            conda-forge/linux-64::libtiff-4.1.0-hc7e4089_6
  libwebp-base       conda-forge/linux-64::libwebp-base-1.1.0-h516909a_3
  llvm-openmp        conda-forge/linux-64::llvm-openmp-10.0.1-hc9558a2_0
  lz4-c              conda-forge/linux-64::lz4-c-1.9.2-he1b5a44_3
  numpy              conda-forge/linux-64::numpy-1.16.5-py37h95a1406_0
  olefile            conda-forge/noarch::olefile-0.46-py_0
  onnx               conda-forge/linux-64::onnx-1.6.0-py37he1b5a44_0
  pillow             conda-forge/linux-64::pillow-7.2.0-py37h718be6c_1
  protobuf           conda-forge/linux-64::protobuf-3.9.2-py37he1b5a44_1
  python_abi         conda-forge/linux-64::python_abi-3.7-1_cp37m
  singa              nusdbsystem/linux-64::singa-3.0.0-cpu_py37
  singa-cpu          nusdbsystem/linux-64::singa-cpu-3.0.0-py37
  six                conda-forge/noarch::six-1.15.0-pyh9f0ad1d_0
  tqdm               conda-forge/noarch::tqdm-4.48.2-pyh9f0ad1d_0
  wrapt              conda-forge/linux-64::wrapt-1.12.1-py37h8f50634_1
  zstd               conda-forge/linux-64::zstd-1.4.5-h6597ccf_2

The following packages will be UPDATED:

  libgcc-ng           pkgs/main::libgcc-ng-9.1.0-hdf63c60_0 --> conda-forge::libgcc-ng-9.3.0-h24d8f2e_16
  openssl              pkgs/main::openssl-1.1.1g-h7b6447c_0 --> conda-forge::openssl-1.1.1g-h516909a_1

The following packages will be SUPERSEDED by a higher-priority channel:

  _libgcc_mutex           pkgs/main::_libgcc_mutex-0.1-main --> conda-forge::_libgcc_mutex-0.1-conda_forge
  ca-certificates    pkgs/main::ca-certificates-2020.7.22-0 --> conda-forge::ca-certificates-2020.6.20-hecda079_0
  certifi               pkgs/main::certifi-2020.6.20-py37_0 --> conda-forge::certifi-2020.6.20-py37hc8dfbb8_0


Proceed ([y]/n)? y


Downloading and Extracting Packages
singa-cpu-3.0.0      | 4 KB      | #################################################################################### | 100%
singa-3.0.0          | 22.2 MB   | #################################################################################### | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
(sc37) user@xgpe3:~$ python -c "from singa.module import Module"
(sc37) user@xgpe3:~$
```

Is this a bug or just the way it goes?

**Comment by nudles** [comment]
Hi, it is renamed to [Model](https://github.com/apache/singa/blob/dev/python/singa/model.py#L103).

**Comment by tinyAdapter** [comment]
Thanks. By the way, does this mean that `singa-cpu` package is outdated?

**Comment by chrishkchris** [comment]
We have released SINGA version 3.1, you can use the new package that already solved the problem
